
---
float64 total_weight
